uber/models/admin.py

```diff
@@ -19,12 +19,19 @@
 
 class AdminAccount(MagModel):
     attendee_id = Column(UUID, ForeignKey('attendee.id'), unique=True)
-    hashed = Column(UnicodeText)
+    hashed = Column(UnicodeText, private=True)
     access = Column(MultiChoice(c.ACCESS_OPTS))
 
     password_reset = relationship(
         'PasswordReset', backref='admin_account', uselist=False)
 
+    api_tokens = relationship('ApiToken', backref='admin_account')
```

Comment:
This might be because I'm reviewing this during the presentations, but I'm not exactly grokking the relationship between admin accounts and API tokens. Do you have to have an admin account to get token access?

Reply:
That's correct! Not only do you need an admin account, you also need to be granted API access for your admin account. And if someone removes your API access, it will automatically revoke all of your tokens. I wanted to be able to easily trace changes made through the API to a particular person to maintain accountability.